digdag-cli/src/main/java/io/digdag/cli/client/ClientCommand.java

```diff
@@ -102,6 +105,8 @@ protected DigdagClient buildClient(boolean checkServerVersion)
             endpoint = props.getProperty("client.http.endpoint", DEFAULT_ENDPOINT);
         }
 
+        disableDirectDownload = Boolean.parseBoolean(props.getProperty("client.http.disable_direct_download", DEFAULT_DISABLE_DIRECT_DOWNLOAD));
```

Comment:
Now you can move this line inside `buildClient` method so that you don't have to add additional argument to `buildClient` method. I don't have much preference since buildClient is a private static method. But is it intentional to keep disableDirectDownload here?

Reply:
Got it. We don't need to keep the member `disableDirectDownload`.
I will remove it and move the line to `buildClient`

Reply:
Fixed it.